.rubocop.yml

```diff
@@ -46,22 +46,16 @@ Style/SymbolArray:
 Style/HashSyntax:
   EnforcedStyle: ruby19_no_mixed_keys
 
-# This cop no longer recognized.
-# has_key? and has_value? are far more readable than key? and value?
-#Style/DeprecatedHashMethods:
-#  Enabled: false
-
-# Options: format, sprintf, percent
-#Style/FormatString:
-#  EnforcedStyle: false
+# Frozen string literal: http://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/FrozenStringLiteralComment
+Style/FrozenStringLiteralComment:
+  Enabled: false
```

Comment:
`frozen_string_literal: true` at the top of the file makes all the strings immutable. It was introduced in Ruby 2.3 to prepare for Ruby 3, where all strings are going to be immutable by default. You are still able to override that default. The release for Ruby 3 looks like it's still [a few years away](https://blog.heroku.com/ruby-3-by-3) which means we might actually run into more issues with 3rd party tools that we use that are expecting mutable strings, because we don't really know how many people have actually adopted it yet. So I think we're good if we tell rubocop to ignore it